CONTRIBUTING.md

```diff
@@ -96,11 +95,9 @@ The subject contains succinct description of the change:
 * no dot (.) at the end
 
 ### Body
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
+Wrap the body at 72 characters whenever possible (for example, don't modify backtraces to follow this rule). Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
```

Comment:
Not really, no. To me it would seem like backtrace is kinda on the opposite spectrum of what commit messages are - i.e. they're docs/bookkeeping.

I.e. while a snip of code possibly could be to some degree understandable even to people who can't program, I'd say that backtrace is not understandable even to large pool of people who can "program". All these HTML/js coders...

Check, I can code in Rust, and even then I don't really understand backtrace output.